java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -180,6 +180,12 @@ private predicate taintPreservingArgumentToMethod(Method method, int arg) {
     or
     method.hasName(["nCopies", "singletonMap"]) and arg = 1
   )
+  or
+  method.getDeclaringType().hasQualifiedName("java.util", "Arrays") and
+  (
+    method.hasName(["copyOf", "copyOfRange", "deepToString", "spliterator", "stream", "toString"]) and
```

Comment:
What is the general convention regarding `Object.toString()` of this project? It could be a taint step if it is overridden and includes the values of the arguments, but if it is not overridden this would cause false positives.

So if this considers `deepToString` and `toString` as taint steps, `Object.toString()` should probably be considered as well.

Reply:
Let's exclude "deepToString" and "toString" for now.

Reply:
OK